### ISSUE TITLE
AWS Account ID rule false positive

### ISSUE THREAD

**Issue by azu** [secretlint/secretlint#855]
AccountID is a just generatl regexp.
It cause false positve in sometimes.

We want to make change default rule as follows.
```
      "id": "@secretlint/secretlint-rule-preset-recommend",
      "rules": [
        {
          "id": "@secretlint/secretlint-rule-aws",
          "allowMessageIds": ["AWSAccountID", "AWSAccessKeyID"]
        }
      ]
```
